services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -118,7 +119,7 @@ boolean someBrokerWithoutLeader() {
      * @return -1 if partitions do not all have the same replication factor possibly indicating that the topic is
      * undergoing maintenance
      */
-    int replicationFactor() {
+    int currentReplicationFactor() {
```

Comment:
Would it be better to change _replicationFactor to `_configuredReplicationFactor`, and use `replicationFactor()` instead of `currentReplicationFactor()`? I don't think `current` tells much useful information. It is not `current` vs. `configured` or `current` vs. `previous`. It is `actual` vs. `configured`.